engine/apps/user_management/models/user.py

```diff
@@ -75,25 +75,46 @@ def sync_for_team(team, api_members: list[dict]):
 
     @staticmethod
     def sync_for_organization(organization, api_users: list[dict]):
+        from apps.base.models import UserNotificationPolicy
+
         grafana_users = {user["userId"]: user for user in api_users}
         existing_user_ids = set(organization.users.all().values_list("user_id", flat=True))
 
-        # create missing users
-        users_to_create = tuple(
-            User(
-                organization_id=organization.pk,
-                user_id=user["userId"],
-                email=user["email"],
-                name=user["name"],
-                username=user["login"],
-                role=LegacyAccessControlRole[user["role"].upper()],
-                avatar_url=user["avatarUrl"],
-                permissions=user["permissions"],
-            )
-            for user in grafana_users.values()
-            if user["userId"] not in existing_user_ids
-        )
-        organization.users.bulk_create(users_to_create, batch_size=5000)
+        users_to_create = []
+        policies_to_create = []
+        for user in grafana_users.values():
+            if user["userId"] not in existing_user_ids:
+                user = User(
+                    organization_id=organization.pk,
+                    user_id=user["userId"],
+                    email=user["email"],
+                    name=user["name"],
+                    username=user["login"],
+                    role=LegacyAccessControlRole[user["role"].upper()],
+                    avatar_url=user["avatarUrl"],
+                    permissions=user["permissions"],
+                )
+                users_to_create.append(user)
+                policies_to_create.append(
+                    UserNotificationPolicy(
+                        user=user,
+                        step=UserNotificationPolicy.Step.NOTIFY,
+                        notify_by=settings.EMAIL_BACKEND_INTERNAL_ID,
+                        order=0,
+                    ),
+                )
+                policies_to_create.append(
+                    UserNotificationPolicy(
+                        user=user,
+                        step=UserNotificationPolicy.Step.NOTIFY,
+                        notify_by=settings.EMAIL_BACKEND_INTERNAL_ID,
+                        order=0,
+                        important=True,
+                    ),
+                )
```

Comment:
maybe dedup this creation logic (duplicates [this](https://github.com/grafana/oncall/pull/3064/files#diff-6e8b2e38e6eee048bf3817babbd042b94d5752f43b20cf300509e77c1c40aa50)) by creating a method on `User` or `UserNotificationPolicy` model (whichever you feel is more appropriate)